src/components/application_manager/src/mobile_message_handler.cc

```diff
@@ -104,7 +104,7 @@ application_manager::Message* MobileMessageHandler::HandleIncomingMessageProtoco
   }
   LOG4CXX_DEBUG(logger_, "Incoming RPC_INFO: " <<
                          (out_message->connection_key() >> 16) <<", "<<
-                         GetMessageTypes[out_message->type()] <<", "<<
+                         //GetMessageTypes[out_message->type()] <<", "<<
```

Comment:
My thoghts about the commented code. We need to remove it at all. As we use git, so this changes can be repaired at any time